src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -39,9 +35,37 @@ public CaseTransformationResponse transformExceptionRecordToCase(Map<String, Obj
     }
 
     private Appeal buildAppealFromData(Map<String, Object> pairs) {
+
+        Appellant appellant = null;
+
+        if (hasPerson(pairs, "person2")) {
+            Appointee appointee = null;
+            if (hasPerson(pairs, "person1")) {
+                appointee = Appointee.builder()
```

Comment:
you could have extracted the common code to a method ?

Reply:
All 3 common bits of code in this method are actually slightly different. Can you see a way to extract them?